src/dune_engine/package.mli

```diff
@@ -120,10 +120,15 @@ module Info : sig
 
   val maintainers : t -> string list option
 
+  (** Default package info (e.g., for project initialization ) *)
+  val default : t
```

Comment:
It feels more like a "sample" than a "default" to me. I'd expect "default" to have all fields set to `None` for instance.

Reply:
Good call there. I've renamed it to "example".